acceptance/bundle/git-permerror/output.txt

```diff
@@ -16,6 +17,7 @@ Found 1 error
 Exit code: 1
 
 >>> $CLI bundle validate -o json
+Warn: failed to read .git: unable to load repository specific gitconfig: open config: permission denied
```

Comment:
Is the warning printed to stdout or stderr here? If stdout then it will be a breaking changing for people relying on JSON output here

Reply:
stderr